src/webui/webui.cpp

```diff
@@ -147,6 +168,18 @@ void WebUI::configure()
     }
 }
 
+void WebUI::setError(const QStringView message)
```

Comment:
```suggestion
void WebUI::setError(const QString &message)
```
Now I wonder what benefit does a view brings. It doesn't seem useful when there is implicit sharing where you can avoid a deep copy most of the time.

Reply:
> Now I wonder what benefit does a view brings.

Seems I've just misused it. `QStringView` is useful when the function isn't intended to store provided string but to process/parse it in some way.